<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonAddNoise">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add noise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterImage">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Box Filter Image</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ImageWithNoise">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ImageWithFilter">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSaP">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add SaP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddGN">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add GN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterImage_Gaus">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>480</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>G Filter Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterImage_Median">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>M Filter Image</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="deepBox">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>450</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>480</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>480</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sigma</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinTimes">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>510</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>510</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Times</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinKernel">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>510</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>510</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterImage_Median_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>510</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>M Filter Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
